index.html

```diff
@@ -6,91 +6,101 @@
-            <div class="workdescription">
-                <h2>Tonic</h2>
-                <div class="details"> <span id="canopy">CANOPY</span> <img src="images/Count.png" alt="greydot"> <span
-                        class="pair" id="wider">Back End Dev</span> <img src="images/Count.png" alt="greydot"> <span
-                        class="pair">2015</span> </div>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul>
-                    <li class="one">HTML</li>
-                    <li class="two">CSS</li>
-                    <li class="three">Javascript</li>
-                </ul> <button class="seeproject">See project</button>
-            </div>
-        </div> <!-- CARD 4 -->
-        <div class="card">
-            <div class="cardimage"> <img src="images/Snapshoot(4).png" alt=""> </div>
-            <div class="workdescription">
-                <h2>Multi- Post Stories</h2>
-                <div class="details"> <span id="canopy">CANOPY</span> <img src="images/Count.png" alt="greydot"> <span
-                        class="pair" id="wider">Back End Dev</span> <img src="images/Count.png" alt="greydot"> <span
-                        class="pair">2015</span> </div>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul>
-                    <li class="one">HTML</li>
-                    <li class="two">CSS</li>
-                    <li class="three">Javascript</li>
-                </ul> <button class="seeproject">See project</button>
+    <div class="greysection">
+        <header> <a href="#" class="logo">Nasir</a>
+            <nav class="button"><img src="images/Union.png" alt="button">
+
+            </nav>
+        </header>
+        <main id="main-section">
+            <section class="about">
+                <h1>Hi, I am Nasir.<br> Glad to see <br> you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me. </p> <a href="#">LET'S CONNECT</a>
+                <ul class="socialmedia">
+                    <li class="twitter"><a href="#"><img src="images/twitter.png" alt="twitterlogo"></a></li>
+                    <li class="linkedin"><a href="#"><img src="images/linkedIn.png" alt="linkedinlogo"></a></li>
+                    <li class="medium"><a href="#"><img src="images/medium.png" alt="mediumlogo"></a></li>
+                    <li class="github"><a href="#"><img src="images/github.png" alt="githublogo"></a></li>
+                    <li class="angellist"><a href="#"><img src="images/angellist.png" alt="angellistlogo"></a></li>
+                </ul>
+            </section>
```

Comment:
**OPTIONAL** - Please note that on bigger mobile screen the background image for this section is not responsive with the screen. I suggest you use the `background-size: cover` property in CSS to fix this 😄 

Your Project:
![o6o6](https://user-images.githubusercontent.com/73607512/236167391-573a3103-5dc9-4bb9-b797-e160e8065a34.PNG)

Reply:
@mckent05 Noted with thanks 👌🏾